README.md

```diff
@@ -14,7 +14,8 @@ Requirements
 Installation
 ------------
 
-Install the [Symfony client][4] binary and run this command:
+[Download Symfony][4] to install the `symfony` binary in your computer and run
```

Comment:
Is it not « on your computer » ?

Unsure tbh

Reply:
"in your computer" -> 250 million Google results
"on your computer" -> 285 million Google results

So, "on" is the winner here.